src/api/Login.js

```diff
@@ -0,0 +1,35 @@
+import {Axios} from  './Axios';
+
+
+export const AxiosLogin = (data, loginToggle, goMain ) => {
+
+    const LoginSuccess = (accessToken) => {
+        window.localStorage.setItem("accessToken", JSON.stringify(accessToken))
+        const LocalAccessToken = JSON.parse(localStorage.getItem('accessToken'));
+
+        if (LocalAccessToken)
+        {
+            goMain()
+            loginToggle();
+        }
+    }
+
+    Axios.post(`/api/auth/login`, {
+        id: data.id,
+        password: data.password,
+    })
+        .then((res) => {
+            console.log(res.data)
+            LoginSuccess(res.data.accessToken)
+        })
+
+        .catch((error) => {
+            console.log(error)
+            if (error.response && error.response.data && error.response.data.message) {
+                error.response.data.message.map((message) => alert(message));
+
+            } else {
+                console.log("에러", error);
+            }
+        })
+}
```

Comment:
해당 에러 반환문을 보면... 어차피 error.response.data.message가 있으려면 error.response는 있을 수 밖에 없습니다.
상위 속성이니까요

어차피 map하기 위한 데이터가  error.response.data.message 인데
```suggestion
        .catch((error) => {
            console.log(error)
            if (error.response.data.message) {
                error.response.data.message.map((message) => alert(message));
            } else {
                console.log("에러", error);
            }
        })
}
```
라고만 작성해도 되지 않을까요?

Reply:
추가적으로 옵셔널 체이닝이라는 좋은 친구가 있습니다. 
```suggestion
        .catch((error) => {
            console.log(error)
            if (error.response?.data?.message) {
                error.response.data.message.map((message) => alert(message));

            } else {
                console.log("에러", error);
            }
        })
}
```